src/main/java/com/google/sps/data/Post.java

```diff
@@ -36,27 +38,63 @@
     private String location = "";
     private double lat = 0.0;
     private double lng = 0.0;
-    private String numberOfPeopleItFeeds = "";
+    private int numberOfPeopleItFeeds = 0;
     private String typeOfFood = "";
     private String description = "";
     private String collegeId = "";
     private int timeSort = 0;
 
+    public boolean valid = true; // If false, the post shouldn't be saved - it might have malicious data.
+
     /* Fill in the important Post details from the POST request. */
     public void requestToPost(HttpServletRequest request, String collegeId) {
-        organizationName = request.getParameter("organizationName");
-        month = Integer.parseInt(request.getParameter("month")) - 1; // Months are indexed at 0.
-        day = Integer.parseInt(request.getParameter("day"));
-        startHour = Integer.parseInt(request.getParameter("startHour"));
-        startMinute = Integer.parseInt(request.getParameter("startMinute"));
-        endHour = Integer.parseInt(request.getParameter("endHour"));
-        endMinute = Integer.parseInt(request.getParameter("endMinute"));
-        location = request.getParameter("location");
-        lat = Double.parseDouble(request.getParameter("lat"));
-        lng = Double.parseDouble(request.getParameter("lng"));
-        numberOfPeopleItFeeds = request.getParameter("numberOfPeopleItFeeds");
-        typeOfFood = request.getParameter("typeOfFood");
-        description = request.getParameter("description");
+        String orgNameUnparsed = request.getParameter("organizationName");
+        String monthUnparsed = request.getParameter("month");
+        String dayUnparsed = request.getParameter("day");
+        String startHourUnparsed = request.getParameter("startHour");
```

Comment:
Is it possible to change the indentation from 4 spaces to 2 spaces to match [Google's style for Java](https://google.github.io/styleguide/javaguide.html#s4.2-block-indentation)?

Reply:
Fixed!